intake_netcdf/__init__.py

```diff
@@ -70,17 +77,17 @@ def _get_schema(self):
 
     def read(self):
         self._load_metadata()
-        return self._ds
+        return self._ds.load()
 
     def read_chunked(self):
-        raise Exception('read_chunked not supported for xarray containers.')
+        self._load_metadata()
+        return self._ds
 
     def read_partition(self, i):
         raise Exception('read_partition not supported for xarray containers.')
 
     def to_dask(self):
-        self._load_metadata()
-        return self._ds.to_dask_dataframe()
+        return self.read_chunked()
```

Comment:
`read_chunked` is will return the dataset as a lazy container, which I believe is also what we want for the `to_dask` case.

Reply:
Agree, this should just return the xarray directly (which points to dask arrays); this would be `self._ds` if you agree that read_chunked should be an exception.